pkg/controller/service.go

```diff
@@ -71,10 +72,11 @@ func (s *Service) Run(ctx context.Context) error {
 	port := s.Options.Config.Server.Port
 	healthProbePort := *s.Options.Config.WorkerServer.Port
 	mgr, err := ctrl.NewManager(s.Options.K8sConfig, ctrl.Options{
-		Logger:                 logger,
-		CertDir:                s.TLSCertDir,
```

Comment:
So, do you know how the certificates should be configured now that this option has changed? The certificates are needed outside of test scenarios.

Reply:
There is also this setting: https://github.com/radius-project/radius/pull/7498/files#diff-b31c03e05358c6faab6e77c96909e14e14eeb0dc09412ede3329625adc197fa9R85. Can we sync offline on this one? Trying to understand where else we can need this property.